test/Transformers.jl

```diff
@@ -43,27 +42,30 @@ infos = info_dict(selector)
 @test infos[:output_scitype] == MLJBase.Table(Scientific)
 
 #### FeatureSelectorRule ####
-using DataFrames
 X = (name       = categorical(["Ben", "John", "Mary", "John"], ordered=true),
      height     = [1.85, 1.67, 1.5, 1.67],
      dummy     = [1.0, 1.00, 1.0, 1.0],
      favourite_number = categorical([7, 5, 10, 5]),
      age        = [23, 23, 14, 23],
      gender     = categorical(['M', 'M', 'F', 'M']))
 
+std(X[:height])
 
-function var_rule(n,t,s)
-    return s <: Continuous ? (std(X[n])>0.1 ? true : false) : true
+mutable struct StdRule <: MLJModels.SelectorRule
+     threshold::Float64
 end
+(sr::StdRule)(X,name,type,scitypes) =  scitypes <: Continuous ? (std(X[name])>sr.threshold ? true : false) : true
+const StdSelector = MLJModels.FeatureSelectorRule{StdRule}
+StdSelector(;threshold=0.2)= StdSelector(StdRule(threshold))
+fsr=StdSelector()
+fsr_fit,=fit(fsr,1,X)
+
```

Comment:
add tests for the rule and the structure of FeatureSelectorRule testing fields etc + also test creation of rules + what happens if your rule does not meet the specs, is there an appropriate error message? add one